data/quickstarts/create-ml-pipeline-quickstart.yaml

```diff
@@ -0,0 +1,92 @@
+kind: ConsoleQuickStart
+metadata:
+  name: create-ml-pipeline
+  annotations:
+    opendatahub.io/categories: 'Getting started,ML Pipelines'
+spec:
+  displayName: Creating an ML Pipeline
+  appName: ml-pipelines
+  durationMinutes: 5
+  icon: 'images/kubeflow.svg'
+  description: This quick start will walk you through creating an ML Pipeline
```

Comment:
The descriptions usually take the form "Do X to achieve Y" - in this case, maybe "Create a simple pipeline that automatically runs tasks in a machine learning deployment workflow" or similar would work?

Reply:
I like that simple and concise description, updating to it.  Hope you don't mind me stealing the verbage :)